File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/card/GetCardDataResponse.java
package com.lattis.ellipse.data.network.model.response.card;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 7/26/17.
 */

public class GetCardDataResponse {

    @SerializedName("id")
    private int id;

    @SerializedName("user_id")
    private int user_id;

    @SerializedName("stripe_net_profile_id")
    private String stripe_net_profile_id;

    @SerializedName("stripe_net_payment_id")
    private String stripe_net_payment_id;

    @SerializedName("is_primary")
    private boolean is_primary;

    @SerializedName("type_card")
    private String type_card;

    @SerializedName("cc_no")
    private String cc_no;

    @SerializedName("exp_month")
    private int exp_month;

    @SerializedName("exp_year")
    private int exp_year;

    @SerializedName("fingerprint")
    private String fingerprint;

    @SerializedName("cc_type")
    private String cc_type;

    @SerializedName("created_date")
    private int created_date;

    @SerializedName("last_updated")
    private String last_updated;

    public int getId() {
        return id;
    }

    public int getUser_id() {
        return user_id;
    }

    public String getStripe_net_profile_id() {
        return stripe_net_profile_id;
    }

    public String getStripe_net_payment_id() {
        return stripe_net_payment_id;
    }

    public boolean getIs_primary() {
        return is_primary;
    }

    public String getType_card() {
        return type_card;
    }

    public String getCc_no() {
        return cc_no;
    }

    public int getExp_month() {
        return exp_month;
    }

    public int getExp_year() {
        return exp_year;
    }

    public String getFingerprint() {
        return fingerprint;
    }

    public String getCc_type() {
        return cc_type;
    }

    public int getCreated_date() {
        return created_date;
    }

    public String getLast_updated() {
        return last_updated;
    }






}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/card/GetCardDataResponse.kt
package com.lattis.data.entity.response.card

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 7/26/17.
 */

class GetCardDataResponse {

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("user_id")
    val user_id: Int = 0

    @SerializedName("stripe_net_profile_id")
    val stripe_net_profile_id: String? = null

    @SerializedName("stripe_net_payment_id")
    val stripe_net_payment_id: String? = null

    @SerializedName("is_primary")
    val is_primary: Boolean = false

    @SerializedName("type_card")
    val type_card: String? = null

    @SerializedName("cc_no")
    val cc_no: String? = null

    @SerializedName("exp_month")
    val exp_month: Int = 0

    @SerializedName("exp_year")
    val exp_year: Int = 0

    @SerializedName("fingerprint")
    val fingerprint: String? = null

    @SerializedName("cc_type")
    val cc_type: String? = null

    @SerializedName("created_date")
    val created_date: Int = 0

    @SerializedName("last_updated")
    val last_updated: String? = null

    @SerializedName("card_id")
    val card_id: String? = null
}
